tests/listobjects/listobjects.go

```diff
@@ -70,70 +70,73 @@ func runTests(t *testing.T, schemaVersion string, client ListObjectsClientInterf
 	require.NoError(t, err)
 
 	ctx := context.Background()
+	t.Run("runListObjectsTests"+schemaVersion, func(t *testing.T) {
```

Comment:
This adds a lot of redundant nesting. I think it'd be more preferrable to make changes in `tests/check/check_test.go`

Namely,

```
func TestCheckMemory(t *testing.T) {
    RunAll(t, "memory")
}

func RunAll(t *testing.T, engine string) {
    t.Run("Check", ...)
   t.Run("BadAuthModelID", ...)
}

func checkTest(t *testing.T, engine string) {
    ...
    t.Run("SchemaVersion1_1", ...)
    t.Run("SchemaVersion1_0", ...)
}
```

We should strive to be consistent with the composition in `pkg/server/test` and `pkg/server/server_test.go` as well where we can.